frontend/fingertips-frontend/.gitignore

```diff
@@ -12,6 +12,7 @@
 
 # testing
 /coverage
+*.png
```

Comment:
I don't think we want to ignore all PNG files in the entire frontend directory. What if we want to use a PNG in the actual app somewhere?